.github/workflows/bump.yml

```diff
@@ -26,22 +26,34 @@ jobs:
     strategy:
       matrix:
         os:
-          - ubuntu-latest
+          - ubuntu-16.04
```

Comment:
might be good to leave a comment with 'why' on this one. i thought 16 was getting phased out soon?